### PR TITLE
Feat product listing page

### PR DESCRIPTION
**Pull request by anshul-bheniye**
Product listing page
preview

### CODE REVIEW

File: styles/plpstyle.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
+/* Filter  */
+.custom-size{
+    width: 12.5rem;
+    height: 22rem;
+}
+
+.custom-border{
+    border-right: 0.188rem solid;
```
Comment (SanjayJatti) [inline styles/plpstyle.css:9]: I don't understand why border-right : 0.188rem exactly?

**Review by SanjayJatti — COMMENTED** [review]
You should add gif or screenshot of changes you have done to help reviewer.

File: styles/plpstyle.css (hunk 1 above)
Comment (anshul-bheniye) [reply styles/plpstyle.css:9]: 3px = 0.188 rem Avoiding px only using rems

Comment (SanjayJatti) [reply styles/plpstyle.css:9]: For size less than 10px you can use pixel also. No problem.

**Review by SanjayJatti — APPROVED** [review]

File: styles/plpstyle.css (hunk 1 above)
Comment (anshul-bheniye) [reply styles/plpstyle.css:9]: okay

Comment (Kavita539) [inline styles/plpstyle.css:9]: suggestion : No need to convert this in rem, you can use px for values less than 10px

**Review by Kavita539 — COMMENTED** [review]
Everything looks good!

File: styles/plpstyle.css (hunk 1 above)
Comment (anshul-bheniye) [reply styles/plpstyle.css:9]: Okay.

**Review by Kavita539 — APPROVED** [review]

**Pull request merged**